_IBC/kima-osmosis.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "kima",
+    "client_id": "07-tendermint-0",
+    "connection_id": "connection-0"
+  },
+  "chain_2": {
+    "chain_name": "osmosis",
+    "client_id": "07-tendermint-0",
```

Comment:
this should be `07-tendermint-9`

full context:

```json
Chain ID: kima_testnet
Counterparty Chain ID: osmo-test-5
{
  "client_id": "07-tendermint-0",
  "connection_id": "connection-0",
  "counterparty_client_id": "07-tendermint-3979",
  "counterparty_connection_id": "connection-3439",
  "channel_id": "channel-0",
  "counterparty_channel_id": "channel-8847"
}
```

Reply:
updated thank you